lib/Operations/Structured.coffee

```diff
@@ -232,6 +245,14 @@ module.exports = ()->
     push: (content)->
       @insertAfter @end.prev_cl, [content]
 
+    insertAfterHelper: (root, content)->
```

Comment:
Oops, shouldn't be here. I'm updating the PR, until it's mature, I'm cancelling it.